### DISCUSSION TITLE
Unable to authenticate

### DISCUSSION THREAD

**Discussion by Hr46ph** [hashicorp/packer-plugin-vsphere#638] [Community Help]
Issues on GitHub are intended to be related to bugs or feature requests, so we
recommend using our other community resources instead of asking here if you
have a question.

- Packer Guides: https://www.packer.io/guides
- Discussion List: https://groups.google.com/group/packer-tool
- Any other questions can be sent to the packer section of the HashiCorp
  forum: https://discuss.hashicorp.com/c/packer
- Packer community links: https://www.packer.io/community

**Comment by Hr46ph** [comment]
Did I post that? 🤣 

Never meant to. When I read about the other sources for regular questions I mean to cancel.

Sorry 😉

**Comment by tenthirtyam** [comment]
All good! 😆
